Libraries/Network/NativeNetworkingIOS.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import type {NativeResponseType} from './XMLHttpRequest';
+
+export interface Spec extends TurboModule {
+  +sendRequest: (
+    query: {|
+      method: string,
+      url: string,
+      data: {[key: string]: mixed},
+      headers: {[key: string]: mixed},
+      responseType: NativeResponseType,
+      incrementalUpdates: boolean,
+      timeout: number,
+      withCredentials: boolean,
+    |},
+    callback: (requestId: number) => mixed,
+  ) => void;
+  +abortRequest: (requestId: number) => void;
+  +clearCookies: (callback: (result: boolean) => mixed) => void;
```

Comment:
not sure if mix is understood by the codegen, but i would err towards no.

Reply:
@rickhanlonii we need your powers here

Reply:
Why does a callback have a return value?

I don't see any other usages of mixed like this

Reply:
changing to void right now

Reply:
Oh, this change. Please see this comment: https://github.com/facebook/react-native/pull/24892#discussion_r284665079 what do you think?

Reply:
@RSNara could you please advise here? Is using `mixed` for callbacks supported by codegen? Or if we want users to be able to pass async callbacks we should return `Promise<void> | void` instead of just `void`?